src/cloudformation_cli_python_lib/callback.py

```diff
@@ -0,0 +1,45 @@
+import json
+import logging
+from typing import Optional
+from uuid import uuid4
+
+# boto3 doesn't have stub files
+from boto3 import Session  # type: ignore
+
+from .interface import BaseResourceModel, HandlerErrorCode, OperationStatus
+from .utils import KitchenSinkEncoder
+
+LOG = logging.getLogger(__name__)
+
+
+def report_progress(  # pylint: disable=too-many-arguments
+    session: Session,
+    token: str,
+    code: Optional[HandlerErrorCode],
+    status: OperationStatus,
+    current_status: Optional[OperationStatus],
+    model: Optional[BaseResourceModel],
+    message: str,
```

Comment:
would like to have this [match Java](https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/blob/1dd595a016f2071180203ab299aa415c6f17b04d/src/main/java/software/amazon/cloudformation/proxy/CloudFormationCallbackAdapter.java#L51-L56) (and in general, good to disambiguate the different tokens we have), so `bearer_token`, etc.